src/appleseed.studio/mainwindow/pythonconsole/pythoneditor.cpp

```diff
@@ -78,6 +78,8 @@ void PythonEditor::keyPressEvent(QKeyEvent* event)
         insert_spaces(4);
     else
     {
+        if (event->key() == Qt::Key_Return || event->key() == Qt::Key_Enter)
```

Comment:
Yes, I've read it too. It's similar to how I solved it but my solution is simpler. I just unset shift modifier in case shift+enter is pressed